client/containers/Pub/SpubHeader/SpubHeaderField.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Label } from '@blueprintjs/core';
+
+require('./spubHeaderField.scss');
+
+type Props = {
+	asLabel?: boolean;
+	children?: React.ReactNode;
+	title: React.ReactNode;
+	instructions?: React.ReactNode;
+	fullWidth?: boolean;
+	hasMargin?: boolean;
+};
+
+const SpubHeaderField = (props: Props) => {
+	const { asLabel, children, title, instructions, fullWidth = false, hasMargin = true } = props;
```

Comment:
`asLabel` is optional, if unspecified it'll be undefined which is falsy. It isn't given a default.
`fullWidth` is optional, same setup, but it is assigned a default. Is there a subtly I am not seeing, or are these two ways of going about the same thing?

Reply:
The behavior is identical but I agree that I should add an explicit fallback for `asLabel` for consistency.